main.py

```diff
@@ -0,0 +1,97 @@
+import typer
+from rich import print
+
+from waffle_utils.dataset import Dataset
+from waffle_utils.dataset.format import Format
+from waffle_utils.file.io import unzip
+from waffle_utils.file.network import get_file_from_url
+
+app = typer.Typer()
+
+# dataset docs
+name_docs = "Dataset Name"
+root_dir_docs = "Dataset Root Directory. Default to ./datasets"
+
+# coco docs
+coco_file_docs = "COCO json file"
+coco_root_dir_docs = "COCO image root directory"
+
+# split docs
+train_split_ratio_docs = (
+    "train data ratio. val ratio will be set to (1-train_split_ratio)"
+)
+
+# export docs
+format_docs = f"[{', '.join(map(lambda x: x.name, Format))}]"
```

Comment:
What do you think about managing these redundant docstrings in one file.
Will it be over-abstraction?

Give me some idea.

Reply:
It's hard to choose now. Let's proceed as it is and make a decision.

Reply:
If we are going to use this type of docstring template(?) in other scripts as well, I would say it'd be better that we have some isolated docstring handler or manager script. Otherwise, it will be an overengeenring.

Reply:
I will create an issue (#12) to address this problem later.